ios/ItaliaApp.xcodeproj/project.pbxproj

```diff
@@ -13,6 +13,8 @@
 		00C302EA1ABCBA2D00DB3ED1 /* libRCTVibration.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 00C302E41ABCB9EE00DB3ED1 /* libRCTVibration.a */; };
 		00E356F31AD99517003FC87E /* ItaliaAppTests.m in Sources */ = {isa = PBXBuildFile; fileRef = 00E356F21AD99517003FC87E /* ItaliaAppTests.m */; };
 		018DA104690D446AB4939E94 /* libRNInstabug.a in Frameworks */ = {isa = PBXBuildFile; fileRef = B59B6238889E4208B49E5C9D /* libRNInstabug.a */; };
+		0A2B90C3DF6444AEB67B60BD /* Octicons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 40FC08429CCB45218C278983 /* Octicons.ttf */; };
```

Comment:
Since no library or modules is added, are these new resources needed by this PR?